identity-service/src/audiusLibsInstance.js

```diff
@@ -30,7 +31,8 @@ class AudiusLibsWrapper {
           ownerWallet: config.get('relayerPublicKey')
         }
       },
-      isServer: true
+      isServer: true,
+      captchaConfig: { serviceKey: config.get('recaptchaServiceKey') }
```

Comment:
thinking that maybe we should disable this for local dev? or create new keys for local dev?

Reply:
will fail right now since the default is a dummy value of `recaptcha_service_key`

Reply:
perhaps only pass captchaConfig in if recaptchaServiceKey is defined

Reply:
added a check in the middleware instead!